src/components/UI/Text/TimeInput.vue

```diff
@@ -46,7 +46,7 @@
         :maxLength="2"
         :boxStyling="[defaultBoxClass, secondInputInvalidClass]"
         :isDisabled="isDisabled"
-        v-tooltip.bottom="{ value: disabledInputTooltip, disabled: !isDisabled }"
+        v-tooltip="disabledInputTooltip"
```

Comment:
seems like the name for the tooltip variable in this file is not quite accurate.